File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        char[] array = my_string.toCharArray();
        
        for (int i = 0; i < array.length; i++) {
            if (Character.isUpperCase(array[i])) {
                array[i] = Character.toLowerCase(array[i]);
            } else if (Character.isLowerCase(array[i])) {
                array[i] =  Character.toUpperCase(array[i]);
            }
        }
        
        for (int j = 0; j < array.length; j++) {
            answer += array[j];
        }
        
        
        return answer;
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    var answer = '';
    
    for (var c of my_string) {
        if (c === c.toUpperCase()) {
            answer += c.toLowerCase();
        } else if (c === c.toLowerCase()) {
            answer += c.toUpperCase();
        }
    }
    return answer;
}